js/state/contentSettings.js

```diff
@@ -172,6 +172,16 @@ const getContentSettingsFromSiteSettings = (appState) => {
       addContentSettings(contentSettings.referer, hostPattern, '*', 'allow')
     }
   }
+
+  // temporarySiteSettings
```

Comment:
contentSettings don't have to worry about regular vs private. The private session has an overlay pref store that reads from the main profile, but writes to an in-memory store that doesn't transfer to the main profile